app/models/form_item.rb

```diff
@@ -130,8 +131,8 @@ def base_item
     self
   end
 
-  def visible_children
-    sorted_children.select(&:visible?)
+  def visible_and_enabled_children
```

Comment:
Can we keep the name `visible_children` here? For the sake of practical use, I think that is implied. The implementation would stay the same. I could be sold on adding normalization and/or a DB constraint so that visible always implies enabled.

Reply:
Will do [discussed DB constraint on slack and not implementing that]